app/views/content_items/document_collection.html.erb

```diff
@@ -1,25 +1,42 @@
-<%= render 'shared/title_and_translations', content_item: @content_item %>
-<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
-<%= render 'shared/metadata', content_item: @content_item %>
-<%= render 'shared/history_notice', content_item: @content_item %>
-<%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
-
-<div
-  class="grid-row sidebar-with-body"
-  data-module="sticky-element-container"
-  id="contents"
->
-  <% if @content_item.contents.any? %>
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+       context: t("content_item.schema_name.#{@content_item.document_type}", count: 1),
+       title: @content_item.title,
+       average_title_length: "long" %>
+  </div>
+  <% if @content_item.available_translations.length > 1 %>
```

Comment:
We have a partial, `shared/translations` that we can use here - [link](https://github.com/alphagov/government-frontend/blob/master/app/views/shared/_translations.html.erb). It essentially contains all the logic you've put here, but using it instead just makes the view a bit tidier 🙂